aurora/model/aurora.py

```diff
@@ -342,4 +376,6 @@ def configure_activation_checkpointing(self):
     patch_size=10,
     encoder_depths=(6, 8, 8),
     decoder_depths=(8, 8, 6),
+    # One particular static variable requires a different normalisation.
+    surf_stats={"z": (-3.270407e03, 6.540335e04)},
```

Comment:
This was removed on `main`. Sorry for the hassle!

Reply:
Done.